Makefile

```diff
@@ -12,4 +12,10 @@ all: docker
 # QUAY_REGISTRY_USER=<my-username> make docker -e.
 docker:
 	docker build --build-arg NFD_VERSION=$(VERSION) \
+		--build-arg http_proxy=$(http_proxy) \
```

Comment:
I'm not sure if this change is required. Usually developers use software mechanisms to bypass firewall and proxy settings.

Reply:
Do you see that this change would cause some problems? There are developers, like me, who do not use these software mechanisms or do not want to deploy those in every development environment. These proxy settings are ubiquitous all over kubernetes so why couldn't nfd build support these, as well?